### PR TITLE
Openapi spec sentences

### PR DESCRIPTION
**Pull request by emileswarts**
Add tags to indicate upstream APIs for additional context.

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -255,6 +255,40 @@ paths:
...
+                properties:
+                  data:
+                    type: array
+                    minItems: 0
+                    items:
+                      $ref: "#/components/schemas/Sentence"
+                  pagination:
+                    $ref: "#/components/schemas/Pagination"
+        "404":
+          description: Failed to find sentences a person with the provided PNC ID.
```
Comment (chubberlisk) [inline openapi.yml:283]:
```suggestion
          description: Failed to find sentences for a person with the provided PNC ID.
```

File: openapi.yml [hunk 2]
```diff
@@ -743,6 +769,106 @@ components:
...
           example: 2008/0545166T
           description: An identifier from the Police National Computer (PNC)
+    Sentence:
+      type: object
+      properties:
+        dataSource:
+          type: string
+          enum: [NOMIS, NDELIUS]
+          example: NOMIS
+          description: Which upstream API the sentence originates from
```
Comment (chubberlisk) [inline openapi.yml:779]:
```suggestion
          description: Which legacy system the sentence originates from
```

Thinking "upstream API" would be "Prison API" and "legacy system" is "NOMIS"?

File: openapi.yml [hunk 3]
```diff
@@ -743,6 +769,106 @@ components:
...
+          tags: nomis, ndelius
+          type: string
+          example: Young Offender Inst - >=12 mths
+          description: Description of the sentence
+        isActive:
+          tags: nomis, ndelius
+          type: boolean
+          example: true
+          description: Whether the sentence is active
+        isCustodial:
```
Comment (chubberlisk) [inline openapi.yml:796]: Is this missing tags?

File: openapi.yml [hunk 4]
```diff
@@ -743,6 +769,106 @@ components:
...
+          enum: [Hours, Days, Weeks, Months, Years]
+          example: Hours
+          description: Time unit that is used in combination with the duration field
+        terms:
+          tags: nomis
+          type: array
+          minItems: 0
+          items:
+            $ref: "#/components/schemas/Term"
+    Term:
```
Comment (chubberlisk) [inline openapi.yml:828]: Should this be `SentenceTerm`?

**Review by chubberlisk — APPROVED** [review]
🚢

File: openapi.yml (hunk 3 above)
Comment (emileswarts) [reply openapi.yml:796]: Intentionally left this one out. 
The flag exists in nDelius only, we always set this to true for Nomis within our own API.

Feels like tagging both, or only one API in this case would be deceiving.
Thoughts?

Comment (chubberlisk) [reply openapi.yml:796]: Ahhhhh okay, I think let's leave it out!

**Pull request merged**